test/fork/deployment/InitialState.t.sol

```diff
@@ -74,6 +74,11 @@ contract ContractsInitialStateTest is Test, Utilities, DeploymentFixtures {
         );
     }
 
+    function test_strikes_initialState() public view {
+        assertEq(strikes.treeRoot(), bytes32(0));
+        assertEq(keccak256(abi.encodePacked(strikes.treeCid())), keccak256(""));
```

Comment:
Immutables asserts?

Reply:
wdym?

Reply:
Add asserts for the immutable parameters values